### PR TITLE
Update nuget feed link

### LINKED ISSUE #10465: The .NET Framework feed link is broken
[The .NET Framework feed](https://nuget.org/api/v2/curated-feeds/dotnetframework/Packages/) returns HTTP 404.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5cf5eb78-b881-faaa-b112-531f865d4f81
* Version Independent ID: 63b2567c-323f-8d5c-480f-cd6ab5e2400d
* Content: [The .NET Framework and Out-of-Band Releases](https://docs.microsoft.com/en-us/dotnet/framework/get-started/the-net-framework-and-out-of-band-releases)
* Content Source: [docs/framework/get-started/the-net-framework-and-out-of-band-releases.md](https://github.com/dotnet/docs/blob/master/docs/framework/get-started/the-net-framework-and-out-of-band-releases.md)
* Product: **dotnet-framework**
* GitHub Login: @mairaw
* Microsoft Alias: **mairaw**

### LINKED ISSUE #10538: Broken .NET Framework feed link
https://nuget.org/api/v2/curated-feeds/dotnetframework/Packages/

https://github.com/dotnet/docs/blob/f1e3bca622a94e8e2055a5815fa209b4be43c449/docs/framework/get-started/the-net-framework-and-out-of-band-releases.md#L50

https://github.com/dotnet/docs/blob/f1e3bca622a94e8e2055a5815fa209b4be43c449/docs/framework/whats-new/index.md#L31

### PR DESCRIPTION
**Pull request by Potapy4**
Fixes #10465
Fixes #10538

## Summary

Previous [link](https://nuget.org/api/v2/curated-feeds/dotnetframework/Packages) was redirected to [incorrect url](https://www.nuget.org/curated-feeds/microsoftdotnet/Packages) - this is because Nuget moved its API to v2. I changed it to force a redirect to a valid url.

### CODE REVIEW

**Comment by mairaw** [comment]
Thanks for your PR @Potapy4. @vivmishra can you confirm this is indeed what used to be the .NET Framework NuGet feed? Or should we just call the link differently?

File: docs/framework/get-started/the-net-framework-and-out-of-band-releases.md [hunk 1]
```diff
@@ -47,8 +47,8 @@ OOB releases for core common language runtime (CLR) components are delivered thr
 
  To find both prerelease and stable packages, choose **Include Prerelease** in the NuGet Package Manager.
 
- If you want to be notified of stable package releases, subscribe to [the .NET Framework feed](https://nuget.org/api/v2/curated-feeds/dotnetframework/Packages/).
+ If you want to be notified of stable package releases, subscribe to [the .NET Framework feed](https://www.nuget.org/api/v2/curated-feeds/microsoftdotnet/Packages/).
```
Comment (NextTurn) [inline docs/framework/get-started/the-net-framework-and-out-of-band-releases.md]: Tip: Another link is on line 31.

Comment (NextTurn) [inline docs/framework/get-started/the-net-framework-and-out-of-band-releases.md]: Tip: Another link is at docs/docs/framework/whats-new/index.md#L31.

Comment (Potapy4) [reply docs/framework/get-started/the-net-framework-and-out-of-band-releases.md]: @NextTurn thanks!

**Comment by vivmishra** [comment]
@terrajobst

**Comment by NextTurn** [comment]
While waiting on reviews ... You could update the PR title to make it more descriptive.

File: docs/framework/get-started/the-net-framework-and-out-of-band-releases.md (hunk 1 above)
Comment (NextTurn) [inline docs/framework/get-started/the-net-framework-and-out-of-band-releases.md]:
```suggestion
 If you want to be notified of stable package releases, subscribe to [the .NET Framework feed](https://www.nuget.org/api/v2/curated-feeds/microsoftdotnet/).
```

This is taken from [Microsoft .NET Framework NuGet Packages](https://devblogs.microsoft.com/dotnet/p/nugetpackages/) and should be the only Nuget feed.

See also [Nuget Feed Address](https://stackoverflow.com/questions/5578834/nuget-feed-address) on SO.

File: docs/framework/whats-new/index.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ This article summarizes key new features and improvements in the following versi
...
+> The .NET Framework team also releases features out of band with NuGet to expand platform support and to introduce new functionality, such as immutable collections and SIMD-enabled vector types. For more information, see [Additional Class Libraries and APIs](../additional-apis/index.md) and [The .NET Framework and Out-of-Band Releases](~/docs/framework/get-started/the-net-framework-and-out-of-band-releases.md). See a [complete list of NuGet packages](https://blogs.msdn.microsoft.com/dotnet/p/nugetpackages/) for the .NET Framework, or subscribe to [our feed](https://www.nuget.org/api/v2/curated-feeds/microsoftdotnet/Packages/).
```
Comment (NextTurn) [inline docs/framework/whats-new/index.md]: There isn't a list of Nuget packages on that page referenced by the 3rd link.

File: docs/framework/get-started/the-net-framework-and-out-of-band-releases.md (hunk 1 above)
Comment (terrajobst) [reply docs/framework/get-started/the-net-framework-and-out-of-band-releases.md]: We should delete the entire line. We no longer maintain the Microsoft and .NET feed which is why it no longer ships in the default feeds.

The replacement for them is [NuGet's package identity and trust features](https://blog.nuget.org/20170417/Package-identity-and-trust.html).

File: docs/framework/whats-new/index.md (hunk 2 above)
Comment (terrajobst) [inline docs/framework/whats-new/index.md]:
```suggestion
> The .NET team also releases features out of band with NuGet to expand platform support and to introduce new functionality, such as immutable collections and SIMD-enabled vector types. For more information, see [Additional Class Libraries and APIs](../additional-apis/index.md) and [The .NET Framework and Out-of-Band Releases](~/docs/framework/get-started/the-net-framework-and-out-of-band-releases.md). See a [complete list of NuGet packages](https://www.nuget.org/profiles/dotnetframework).
```

**Review by mairaw — APPROVED** [review]
I've applied the changes suggested by @terrajobst and will merge this PR as soon as build finishes successfully. Changes should be live next week.

**Pull request merged**